hibernate-core/src/main/java/org/hibernate/id/enhanced/NoopOptimizer.java

```diff
@@ -38,12 +38,6 @@ public Serializable generate(AccessCallback callback) {
 		// The lastSourceValue field is only accessed by tests,
 		// so this is not a concern.
 		IntegralDataTypeHolder value = callback.getNextValue();
```

Comment:
@sebersole, any idea why NoopOptimizer cycles through sequence values until it's 1?